### PR TITLE
add product data

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: src/pages/Products/ProductListing.jsx [hunk 1]
```diff
@@ -1,57 +1,48 @@
...
+import { Filter } from "../Products/components/Filter";
+import { Navbar } from "./../../components/navbar/Navbar";
+import { Footer } from "./../../components/footer/Footer";
+import { ProductCard } from "./components/ProductCard";
+import axios  from "axios";
+import { useState, useEffect } from "react";
 
 const ProductListing = () => {
+  const [products, setProducts] = useState([]);
+  console.log(products)
```
Comment (AtulPant2704) [inline src/pages/Products/ProductListing.jsx:11]: Remove this console.log

File: src/pages/Products/ProductListing.jsx [hunk 2]
```diff
@@ -1,57 +1,48 @@
...
-                productAuthor={"Saurabh Mukherjea"}
-                productPrice={922}
-                productRating={3}
-              />
-          </div>
+  const productApi = "/api/products";
+  const loadProductData = async () => {
+    try {
+      const response = await axios.get(productApi);
+      setProducts(response.data.products);
```
Comment (AtulPant2704) [inline src/pages/Products/ProductListing.jsx:17]: Check here for response.status, if it is 200 then only set the products

**Review by kart-c — APPROVED** [review]

**Review by AtulPant2704 — APPROVED** [review]
Rest all looks good.

File: src/pages/Products/ProductListing.jsx (hunk 2 above)
Comment (vermasavita) [reply src/pages/Products/ProductListing.jsx]: Done

File: src/pages/Products/ProductListing.jsx (hunk 1 above)
Comment (vermasavita) [reply src/pages/Products/ProductListing.jsx]: Done

**Pull request merged**